### PR TITLE
Host encoding.spec.whatwg.org

### PR DESCRIPTION
**Pull request by foolip**
The fdupes script has been run, but isn't set up as a cron job. It's
probably sufficient to run it once every few years.

### CODE REVIEW

File: debian/marquee/scripts/fdupes [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash -e
+
+# This need https://github.com/tobiasschulz/fdupes for --linkhard support.
```
Comment (annevk) [inline debian/marquee/scripts/fdupes]: needs*

File: debian/marquee/scripts/fdupes [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+fdupes --recurse --linkhard --noprompt /var/www/encoding.spec.whatwg.org
```
Comment (annevk) [inline debian/marquee/scripts/fdupes]: Why not apply this everywhere?

File: debian/marquee/scripts/fdupes (hunk 1 above)
Comment (foolip) [reply debian/marquee/scripts/fdupes]: Fixed.

File: debian/marquee/scripts/fdupes (hunk 2 above)
Comment (foolip) [reply debian/marquee/scripts/fdupes]: It takes longer to run the more files there are, and we wouldn't gain much by it:
```
$ fdupes --recurse --noempty --summarize /var/www/
303 duplicate files (in 35 sets), occupying 4.8 megabytes
```

That's quite meager compared to the gigabytes saved for encoding. And, there's always the chance that there are fdupes bugs or yet-unknown downsides to hardlinking, so I'm being conservative.

**Comment by foolip** [comment]
OK, I added the script to build fdupes and tweaked the one to run it so that it can just be sudo'd. I still didn't make it a cron job, because it'd be work to make it not spammy, not overwrite existing cron entries, etc.

**Pull request merged**